docs/players-and-controllers/squeezebox-receiver.md

```diff
@@ -43,12 +42,37 @@ Before first use the Receiver needs to be connected to your LMS system using the
 9. Restart your Receiver. If you did everything right, the light now turns green, signaling that an IP address has been acquired via DHCP.
 10. Once the Receiver is connected to LMS, the light will turn white and the device shows up in the list of players.
 
-!!! note
-    It may be recommendable to use wired network with the Receiver. There are reports that the wireless connections suffer from poor connection.
+- It may be recommendable to use wired network with the Receiver. There are reports that the wireless connections suffer from poor connection.
```

Comment:
Could you please update your branch to be up to date with `main` to make sure you don't overwrite the few changes I applied?

Reply:
Hold on: I'll simply copy the new content on my end. Thanks!